XenModel/Actions/VM/VMCloneAction.cs

```diff
@@ -40,14 +41,13 @@ public class VMCloneAction : AsyncAction
         protected string _cloneName;
         protected string _cloneDescription;
         public VMCloneAction(VM vm, string name, string description)
-            : base(vm.Connection, string.Format(Messages.CREATEVM_CLONE, name, vm.Name()))
+            : base(vm.Connection, string.Format(Messages.CREATEVM_CLONE, name, vm.NameWithLocation()))

```

Comment:
Is name with location the best for this? Is the location the host? if so why not use name with host?

Reply:
The location is the host.
My rationale for this was:

- Method is there and is used for this purpose, so it's easier to maintain
- I would have add to modify the CREATEVM_CLOSE string resource to take into account a new variable for the `string.Format` call